### ISSUE TITLE
Safari: MegaMenu is missing blue focus outline that is present on FF and Chrome

### ISSUE THREAD

**Issue by bennettcc** [NCIOCPL/cgov-digital-platform#3587]
### Basic Information ###
- Issue: On Safari, when I expand and collapse mega menu, the L1 item does not have a blue focus  frame , but Chrome and FF do
- Desired Outcome:

### Steps to Reproduce ###
1. Click on any mega menu L1 item  on Safari to expand
2.Click the caret  to collapse
3. See how there is no outline, do the same on FF and Chrome, focus is maintained

### Screen Shots ###

![Image](https://user-images.githubusercontent.com/67657582/193886307-3b118dc6-375f-4437-b454-4cc0bb23b43f.png)
![Image](https://user-images.githubusercontent.com/67657582/193886308-1ce2500a-8ed6-449a-b72b-bb35ec616ef2.png)

**Comment by sarinapadilla** [comment]
This behaves exactly as USWDS does between Chrome and Safari. @arcepaul is going to check if this is an accessibility thing; otherwise, we will behave as USWDS does between browsers.
